util/megaquit.m

```diff
@@ -0,0 +1,4 @@
+%NOTEST
```

Comment:
really?  :)  I think megaclear, quit is probably enough.  i wouldn't include this.